page/index.html

```diff
@@ -53,20 +53,22 @@
        }
        
        #game_editor {
-	   width:50%;
+	   width:49%;
+	   margin-left:.1%;
 	   height:100%;
 	   float:left;
 	   display:inline-block;
 	   color: #ddd !important;  
 	   background-color:#222;
```

Comment:
These magic numbers and attribute orderings are not intuitive to me. However, given this is a testing interface that isn't intended to be included in the final library, I don't really mind it. This is OK given the circumstances.

Reply:
If things like this are ever going to matter, the solution is a linter and a one-shot "we went back and re-linted all the code and here it is now" PR. I don't have much of an opinion on choice of linter or on its settings, so feel free to do this if/when it does matter. 

The magic numbers are not intuitive; they are the outcome of the highly precise and recondite process described [here](https://media1.tenor.com/images/614c9b4639a2588383f47e138177da81/tenor.gif?itemid=12014506).

Reply:
The reason the built/runtime files were in the repository is that the Github Pages page can then be an actual, playable demo. It would probably make more sense for that page to be generated from a separate branch with just the built files and this testing interface, and none of the source. To move in this direction, I've scrubbed the generated files for now.

An argument for keeping the interface in master is that not having to switch branches to play around with it in the browser would be nice too.